src/pages/cart/Cart.jsx

```diff
@@ -0,0 +1,57 @@
+import { HorizontalCard } from '../../components/component_index';
+import { cardData } from '../../data/cardData/cardData';
+import './Cart.css';
+
+export const Cart = () => {
+  return (
+    <main class="cart-body">
```

Comment:
You can use className instead of class in jsx.

Reply:
Yup will do that.